tests/Aspire.Hosting.Containers.Tests/FromDockerfileTests.cs

```diff
@@ -18,7 +19,8 @@ namespace Aspire.Hosting.Containers.Tests;
 
 public class WithDockerfileTests(ITestOutputHelper testOutputHelper)
 {
-    [Fact]
+    // Currently we can only run this locally because the CI agents don't have buildkit support enabled.
+    [LocalOnlyFact]
```

Comment:
:|

Reply:
Temporary until we can get buildkit working. Only impacts secret scenarios.

Reply:
Ok it's working again through the power of environment variables.